library/ix-dev/community/chia/values.yaml

```diff
@@ -0,0 +1,34 @@
+image:
+  repository: ghcr.io/chia-network/chia
+  tag: "1.7.0"
+  pullPolicy: IfNotPresent
+
+resources:
+  limits:
+    cpu: 4000m
+    memory: 8Gi
+
+chiaNetwork:
+  chiaPort: 8444
+  farmerPort: 8447
+
+chiaConfig:
+  # farmer-only | harvester | ""
+  service: ""
+  # farmer_address, farmer_port and ca. Only if sevice == harvester
+  farmer_address: ""
+  farmer_port: ""
+  ca: ""
+  full_node_peer: ""
+  additionalEnvs: []
+
+chiaStorage:
+  data:
+    type: ixVolume
+    hostPath: ""
```

Comment:
Confused for this?

Reply:
It's only there for reference on what questions.yaml have as default.
At install it will get overwritten from user entered values.

When common sees type: `ixVolume` will use the `datasetName` (and ignore `hostPath`). to retrieve the correct host path  from `ixVolumes` list.

If the type is `hostPath` it will ignore `datasetName` and use the supplied `hostPath`